apollo-router/tests/integration_tests.rs

```diff
@@ -401,137 +401,6 @@ async fn automated_persisted_queries() {
     assert_eq!(registry.totals(), expected_service_hits);
 }
 
-#[tokio::test(flavor = "multi_thread")]
-async fn persisted_queries() {
-    use hyper::header::HeaderValue;
-    use serde_json::json;
-
-    /// Construct a persisted query request from an ID.
-    fn pq_request(persisted_query_id: &str) -> router::Request {
-        supergraph::Request::fake_builder()
-            .extension(
-                "persistedQuery",
-                json!({
-                    "version": 1,
-                    "sha256Hash": persisted_query_id
-                }),
-            )
-            .build()
-            .expect("expecting valid request")
-            .try_into()
-            .expect("could not convert supergraph::Request to router::Request")
-    }
-
-    // set up a PQM with one query
-    const PERSISTED_QUERY_ID: &str = "GetMyNameID";
-    const PERSISTED_QUERY_BODY: &str = "query GetMyName { me { name } }";
-    let expected_data = serde_json_bytes::json!({
-      "me": {
-        "name": "Ada Lovelace"
-      }
-    });
-
-    let (_mock_guard, uplink_config) = mock_pq_uplink(
-        &hashmap! { PERSISTED_QUERY_ID.to_string() => PERSISTED_QUERY_BODY.to_string() },
-    )
-    .await;
-
-    let config = serde_json::json!({
-        "persisted_queries": {
-            "enabled": true
-        },
-        "apq": {
-            "enabled": false
-        }
-    });
-
-    let mut config: Configuration = serde_json::from_value(config).unwrap();
-    config.uplink = Some(uplink_config);
-    let (router, registry) = setup_router_and_registry_with_config(config).await.unwrap();
-
-    // Successfully run a persisted query.
-    let actual = query_with_router(router.clone(), pq_request(PERSISTED_QUERY_ID)).await;
-    assert!(actual.errors.is_empty());
-    assert_eq!(actual.data.as_ref(), Some(&expected_data));
-    assert_eq!(registry.totals(), hashmap! {"accounts".to_string() => 1});
-
-    // Error on unpersisted query.
-    const UNKNOWN_QUERY_ID: &str = "unknown_query";
-    const UNPERSISTED_QUERY_BODY: &str = "query GetYourName { you: me { name } }";
-    let expected_data = serde_json_bytes::json!({
-      "you": {
-        "name": "Ada Lovelace"
-      }
-    });
-    let actual = query_with_router(router.clone(), pq_request(UNKNOWN_QUERY_ID)).await;
-    assert_eq!(
-        actual.errors,
-        vec![apollo_router::graphql::Error::builder()
-            .message(format!(
-                "Persisted query '{UNKNOWN_QUERY_ID}' not found in the persisted query list"
-            ))
-            .extension_code("PERSISTED_QUERY_NOT_IN_LIST")
-            .build()]
-    );
-    assert_eq!(actual.data, None);
-    assert_eq!(registry.totals(), hashmap! {"accounts".to_string() => 1});
-
-    // We didn't break normal GETs.
```

Comment:
This particular sub-test got lost because the samples files don't let me specify GET requests with checked responses.